File: Verse.vb
Public Class Verse
    Private _VerseNo As Integer
    Private _VerseText As String

    Public Property VerseNo() As Integer
        Get
            Return _VerseNo
        End Get
        Set(ByVal Value As Integer)
            _VerseNo = Value
        End Set
    End Property

    Public Property VerseText() As String
        Get
            Return _VerseText
        End Get
        Set(ByVal Value As String)
            _VerseText = Value
        End Set
    End Property
End Class

File: branches/BibleWebService_trunk_2.0.0.0_CSharpConversion/Verse.cs
namespace BibleWebService
{
    public class Verse
    {
        private int _VerseNo;
        private string _VerseText;

        public int VerseNo
        {
            get { return _VerseNo; }
            set { _VerseNo = value; }
        }

        public string VerseText
        {
            get { return _VerseText; }
            set { _VerseText = value; }
        }
    }
}